src/commands/generate/generators/auth/accounts-manifest.js

```diff
@@ -0,0 +1,28 @@
+let manifest = {
+  sourceFiles: [
+    // Utils
+    { src: '../scaffold/handlers/model.js', target: 'app/models/<ROUTE_NAME>.mjs' },
+    // API
+    { src: '../scaffold/handlers/api-create-list.js', target: 'app/api/<ROUTE_NAME>.mjs' },
+    { src: '../scaffold/handlers/api-delete.js', target: 'app/api/<ROUTE_NAME>/$id/delete.mjs' },
+    { src: '../scaffold/handlers/api-read-update.js', target: 'app/api/<ROUTE_NAME>/$id.mjs' },
+    // Pages
+    { src: '../scaffold/handlers/pages-list.js', target: 'app/pages/<ROUTE_NAME>.mjs' },
+    { src: '../scaffold/handlers/pages-read.js', target: 'app/pages/<ROUTE_NAME>/$id.mjs' },
+  ],
+  elements: [
+    { name: 'CheckBox', package: '@enhance/form-elements', tagName: 'enhance-checkbox' },
+    { name: 'FieldSet', package: '@enhance/form-elements', tagName: 'enhance-fieldset' },
+    { name: 'FormElement', package: '@enhance/form-elements', tagName: 'enhance-form' },
+    { name: 'LinkElement', package: '@enhance/form-elements', tagName: 'enhance-link' },
+    { name: 'PageContainer', package: '@enhance/form-elements', tagName: 'enhance-page-container' },
+    { name: 'SubmitButton', package: '@enhance/form-elements', tagName: 'enhance-submit-button' },
+    { name: 'TextInput', package: '@enhance/form-elements', tagName: 'enhance-text-input' }
+  ],
+  dependencies: [
+    '@begin/validator@0.0.10',
+    'github:enhance-dev/form-elements',
+  ]
+}
```

Comment:
Because of node 18 and aws sdk v3 you will need to add the following dev dependencies.

```
@aws-sdk/client-apigatewaymanagementapi @aws-sdk/client-dynamodb @aws-sdk/client-s3 @aws-sdk/client-sns @aws-sdk/client-sqs @aws-sdk/client-ssm @aws-sdk/lib-dynamodb
```